h/streamer/websocket.py

```diff
@@ -14,6 +14,10 @@
 
 log = logging.getLogger(__name__)
 
+# Mapping incoming message type to handler function. Handlers are added inline
+# below.
+MESSAGE_HANDLERS = {}
```

Comment:
Hmm, I personally think it would be nicer if the handlers are added here, then I don't have to search through the code to see which message types we support. What's the reason to have them inline after the function definition?